src/main/webapp/resources/js/pages/launch/LaunchFiles.jsx

```diff
@@ -0,0 +1,186 @@
+import React from "react";
+import { fetchPipelineSamples } from "../../apis/pipelines/pipelines";
+import AutoSizer from "react-virtualized-auto-sizer";
+import { VariableSizeList as VList } from "react-window";
+import { notification, Space } from "antd";
+import { useLaunch } from "./launch-context";
+import { removeSample } from "../../apis/cart/cart";
+import { SectionHeading } from "../../components/ant.design/SectionHeading";
+import { SampleFilesListItem } from "./files/SampleFilesListItem";
+import { setSelectedSampleFiles } from "./launch-dispatch";
+import { grey3, grey4 } from "../../styles/colors";
+
+/**
+ * React component to display sample files that will be used in the launching
+ * of the pipeline
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function LaunchFiles() {
+  const listRef = React.useRef();
+  const [selected, setSelected] = React.useState();
+  const [height, setHeight] = React.useState(600);
+  const [
+    { acceptsPairedSequenceFiles: paired, acceptsSingleSequenceFiles: singles },
+    dispatch,
+  ] = useLaunch();
+
+  /*
+  State to hold the count of the number of samples currently being displayed by the UI.
+   */
+  const [samples, setSamples] = React.useState();
+
+  /**
+   * Calculate the height of the sample with its files.
+   * @param index
+   * @returns {number}
+   */
+  const getRowHeight = (index) => {
+    if (samples[index].files.length) {
+      return samples[index].files.length * 40 + 50;
+    }
+    return 100;
+  };
+
+  /*
+  Called on initialization.  This gets the samples that are currently in the cart,
+  with their associated files.
+   */
+  React.useEffect(() => {
+    fetchPipelineSamples({
+      paired,
+      singles,
+    })
+      .then((data) => {
+        const firstSelected = [];
+        const firstSamples = [];
+        data.forEach((sample) => {
+          if (sample.files.length) {
+            sample.selected = sample.files[0].identifier;
+            firstSelected.push(sample.selected);
+          }
+          firstSamples.push(sample);
+        });
+
+        setSamples(firstSamples);
+        setSelected(firstSelected);
+      })
+      .catch((message) => notification.error({ message }));
+  }, [paired, singles]);
+
+  /*
+  Called when a user selects a different set of files to run on the sample.
+  Needs to be updated through the reducer so that the correct file ids are
+  set when the pipeline is launched.
+   */
+  React.useEffect(() => {
+    setSelectedSampleFiles(dispatch, selected);
+  }, [dispatch, selected]);
+
+  /*
+  Calculate the div size if the number of samples changes
+   */
+  React.useEffect(() => {
+    if (samples) {
+      let newHeight = 0;
+      for (let i = 0; i < samples.length; i++) {
+        newHeight += getRowHeight(i);
+        if (newHeight > 600) {
```

Comment:
I see that `600` is used a few times in this file when checking/setting the height. Set a const for this and then use the const. `600` is also used in the `Autosizer` height below

Reply:
Updated in 0033f35aefc00eed97837681c5ddef94b8c7f10f